ledger/src/blockstore.rs

```diff
@@ -4132,10 +4132,11 @@ pub fn create_new_ledger(
         blockstore_dir,
     ];
     let output = std::process::Command::new("tar")
+        .env("COPYFILE_DISABLE", "1")
         .args(args)
         .output()
         .unwrap();
-    if !output.status.success() {
+    if !output.stderr.is_empty() || !output.status.success() {
```

Comment:
Does tar exit with a success code but print a message to stderr instead?

Reply:
Yes